src/extension/wps-download/src/main/java/org/geoserver/wps/gs/download/DownloadAnimationProcess.java

```diff
@@ -159,8 +174,18 @@ public RawData execute(
                 new AWTSequenceEncoder(NIOUtils.writableChannel(output.file()), frameRate);
 
         DownloadServiceConfiguration configuration = confiGenerator.getConfiguration();
-        TimeParser timeParser = new TimeParser(configuration.getMaxAnimationFrames());
-        Collection parsedTimes = timeParser.parse(time);
+        int totalTimes;
```

Comment:
Does not represent a count of times anymore. 
```suggestion
        int totalFrames;
```